assets/js/models/pa_message.ts

```diff
@@ -0,0 +1,11 @@
+export interface PaMessage {
+  id: number;
+  sign_ids: string[];
+  priority: number;
+  interval_in_minutes: number;
+  visual_text: string;
+  audio_text: string;
+  start_time: Date;
+  end_time: Date;
+  inserted_at: Date;
```

Comment:
**question:** if this supposed to represent the data returned by the JSON API won't these be `string` values that can be parsed into `Date`s?

Reply:
ah yes good point, thank you!